website/docs/d/cis_cache_settings.html.markdown

```diff
@@ -0,0 +1,60 @@
+---
+layout: "ibm"
+page_title: "IBM: ibm_cis_serve_stale_content"
+sidebar_current: "docs-ibm-datasource-cis-server_stale_content"
+description: |-
+  Get information on IBM Cloud Internet Services Serve Stale Content.
```

Comment:
IBM Cloud Internet Services Cache Settings..
it gives details of all cache settings not only server stale content